src/common/docsEnvironmentController.ts

```diff
@@ -11,6 +11,7 @@ export class DocsEnvironmentController implements EnvironmentController, vscode.
     private _docsRepoType: DocsRepoType;
     private _configurationChangeListener: vscode.Disposable;
     private _userType: UserType;
+    private _enableRealTimeValidation: boolean;

```

Comment:
should this be consistent with the config name?

Reply:
Feel a little bit weird directly using automaticallyEnableRealTimeValidation here, changing to 'enableAutomaticRealTimeValidation' be better I think.